setup.cfg

```diff
@@ -2,9 +2,8 @@
 current_version = 2023.07.01
 parse = (?P<year>2[0-1]\d{2})\.(?P<month>(0\d|1[0-2]))(.(?P<increment>\d{2}))?
 serialize = {year}.{month}.{increment}
-commit_message = "Automatically generated release {new_version}"
 commit = True
-tag = True
+tag = False
```

Comment:
This is necessary because we are amending the commit and then we tag after the amend